### PR TITLE
support wrap/unwrap stETH/wstETH

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/constants/tokenLists/mainnet.tokenlist.ts [hunk 1]
```diff
@@ -245,19 +245,19 @@ export const MAINNET_TOKEN_LIST = {
...
       chainId: 1,
-      address: '0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84',
-      symbol: 'stETH',
-      name: 'stETH (Lido)',
+      address: '0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0',
+      symbol: 'wstETH',
+      name: 'Lido wstETH',
       decimals: 18,
-      logoURI: 'https://assets.coingecko.com/coins/images/13442/large/steth_logo.png',
+      logoURI: 'https://lido.fi/static/images/favicon/apple-touch-icon.png',
```
Comment (namgold) [inline src/constants/tokenLists/mainnet.tokenlist.ts:260]: Why would you changing dis @viet-nv You are using lower quality img

**Comment by viet-nv** [comment]
Closed. We did not get deal with lido team

File: src/constants/tokenLists/mainnet.tokenlist.ts (hunk 1 above)
Comment (viet-nv) [reply src/constants/tokenLists/mainnet.tokenlist.ts:260]: I added this on my branch before you add it, All information were got from cmc then when merging code from main branch, I accepted my code, not yours, haha

Comment (namgold) [reply src/constants/tokenLists/mainnet.tokenlist.ts:260]: :tien-sword:

**Pull request closed without merging**